interface/frontend/src/views/OpenImage.vue

```diff
@@ -1,13 +1,129 @@
 <template>
-  <image-series></image-series>
+  <div class="container">
+    <div class="col-xs-6">
+      <div style="width:512px;height:512px;position:relative;display:inline-block;color:white;"
+        oncontextmenu="return false"
+        class='cornerstone-enabled-image'
+        unselectable='on'
+        onselectstart='return false;'
+        onmousedown='return false;'>
+
+        <div ref="DICOM" style="width:512px;height:512px;top:0px;left:0px; position:absolute;"></div>
+      </div>
+      <p>{{ dicom.imageId}}</p>>
+    </div>
+  </div>
 </template>
 
 <script>
 import ImageSeries from '../components/ImageSeries'
+var cornerstone = require('cornerstone-core')
+var jquery = require('jquery')
```

Comment:
I think usage of jquery should be avoided. According to the code bellow, it seems like it was used to implement a promise. We should be able to use the native browser promise API instead of importing entire JQuery

Reply:
done, was used as a first through, thank you for the review!